CTFd/scoreboard.py

```diff
@@ -5,57 +5,45 @@
 
 scoreboard = Blueprint('scoreboard', __name__)
 
+def get_standings(admin=False, count=-1):
```

Comment:
`-1` doesn't make much sense here. Better to simply go with and check for `None`